test/helpers/mock-cookie-consent-service-helper.ts

```diff
@@ -0,0 +1,20 @@
+import { sinon } from "../utils/test-utils";
+import { CookieConsentServiceInterface } from "../../src/components/common/cookie-consent/types";
+
+export function createMockCookieConsentService(
+  userCookieConsentPreference: string
+): CookieConsentServiceInterface {
+  return {
+    getCookieConsent: sinon.fake(),
+    createConsentCookieValue: sinon.fake.returns({
+      value: JSON.stringify({
+        analytics: userCookieConsentPreference === "true",
+      }),
+      expires: (function (): any {
+        const cookieExpires = new Date();
+        cookieExpires.setFullYear(cookieExpires.getFullYear() + 1);
+        return cookieExpires;
+      })(),
```

Comment:
```suggestion
    expiry: expiryDate
```

Reply:
The `CookieConsentModel` uses expiry but the express Request cookie method uses expires, ideally the model should be updated to align with express.

Reply:
Hi @andrew-moores - thanks for this suggestion. I've just pushed [a new commit](https://github.com/govuk-one-login/authentication-frontend/pull/1588/commits/813241569875bb7b13dae819a7ec33ea4b87ff9c) that does this refactoring